js/viz/docs/docStatic.js

```diff
@@ -129,7 +138,7 @@ var staticMethods = {
     /**
     * @name vizmethods.getPalette
     * @publicName getPalette(paletteName)
-    * @param1 paletteName:string
+    * @param1 palette:string|Array<string>
```

Comment:
Do we really want to mention that it could be of array type?

Reply:
> Do we really want to mention that it could be of array type?

Yes, because you can use your own palette for generate colors. 
```javascript
const colors = DevExpress.viz.getPalette(["red", "blue"]).generateColors(10);
```

> And what about the second parameter (options)?

This parameter isn't needed. You just set palette name (or array of colors).

Reply:
But it would not be possible to get the same colors for gauges